.github/workflows/main.yml

```
@@ -40,20 +39,71 @@ jobs:
       - uses: actions/checkout@v2
         with:
           ref: ${{steps.getcommit.outputs.sha}}
+          fetch-depth: 0
+
+      - uses: octokit/request-action@v2.x
+        if: startsWith(github.event.comment.body, '/run-e2e')
+        name: Getting the files changed in current Pull-Request
+        id: get_files
+        with:
+          route: GET /repos/:repo/pulls/:pull_number/files
+          repo: ${{ github.repository }}
+          pull_number: ${{ github.event.issue.number }}
+        env:
+          GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+
+      - name: Setting up KinD Cluster
+        if: startsWith(github.event.comment.body, '/run-e2e')
+        uses: engineerd/setup-kind@v0.5.0
+        with:
+          version: "v0.7.0"
+
+      - name: Configuring and Testing the Cluster Installation
+        if: startsWith(github.event.comment.body, '/run-e2e')
+        run: |
+          kubectl cluster-info --context kind-kind
+          kind get kubeconfig --internal >$HOME/.kube/config
+          kubectl get nodes
+          kubectl get pods -n kube-system
 
+      - name: Filtering the file paths and building images on the basic of changed files
+        if: startsWith(github.event.comment.body, '/run-e2e')
+        run: |
+            chmod 755 ./.github/filter_and_build.sh
+            ./.github/filter_and_build.sh
+        env:
+          changed_data: ${{steps.get_files.outputs.data}}
+
+      - name: Deploying the litmus-portal for E2E testing
+        if: startsWith(github.event.comment.body, '/run-e2e')
+        run: |
+          kubectl apply -f ./.github/cluster-k8s-manifest.yml
```

can you take the manifest from the litmus-portal directory? because we have to maintain this manifest

ex-

```suggestion
          kubectl apply -f https://raw.githubusercontent.com/litmuschaos/litmus/master/litmus-portal/cluster-k8s-manifest.yml 
```

---

Actually, In original Manifest, the image pull policy is "Always", so it will not take the local images build from PR.
But in this case, we are building images locally which are changed, so i need the local build images to test. So, in manifest, i need the imagepullpolicy as "IfNotPresent". That's why i made another manifest.

---

you can perform the following steps
1. wget  https://raw.githubusercontent.com/litmuschaos/litmus/master/litmus-portal/cluster-k8s-manifest.yml
2. sed -i 's/Always/IfNotPresent/g' cluster-k8s-manifest.yml
3. kubectl apply -f cluster-k8s-manifest.yml

---

Okk @rajdas98 Thanks, Will update it.